kahuna/public/js/image/crop.html

```diff
@@ -15,4 +15,10 @@
     <span class="flex-spacer"></span>
     <span class="image-crop__creator" title="Cropped by {{:: crop.author}} at {{:: crop.date | date:'medium'}}">{{:: crop.author | getInitials}}</span>
   </div>
+  <pinboard-add-button
+    data-type="grid-crop"
+    data-grid-crop-thumbnail="{{:: extremeAssets.smallest | assetFile}}"
+    data-grid-image-id="{{ctrl.image.data.id}}"
+    data-grid-crop-id="{{crop.id}}"
+  ></pinboard-add-button>
```

Comment:
Out of interest can this be self closing???
```suggestion
  />
```

Reply:
we did have it self-closing first, but there was problem with it, can't recall exactly what. Perhaps the linter didn't like it. I do know that IntelliJ guards against self-closing tags...
![image](https://user-images.githubusercontent.com/19289579/109786327-f827bf80-7c04-11eb-9068-5d80bba3cc39.png)
... which seems to be explained in this answer https://stackoverflow.com/a/71836/1868921 (not that we have IE8 users - I hope). Its kind of academic, because in the real DOM it will very quickly become not self-closing once Pinboard's React Portal takes over and adds in HTML within.